src/imagery/i.modis/i.modis.html

```diff
@@ -340,8 +340,3 @@ <h2>AUTHOR</h2>
 
 <h2>SOURCE CODE</h2>
 <p>Available at: <a href="https://github.com/OSGeo/grass-addons/tree/master/grass8/imagery/i.modis">i.modis source code</a> (<a href="https://github.com/OSGeo/grass-addons/commits/grass8/src/imagery/i.modis">history</a>)</p>
```

Comment:
Should the source code links be in here...

Reply:
It might be redundant now that we include the links, but it's easy to find some cases where they aren't generated (or when the link generation won't work anymore). However, for now this link has no value, since it's already wrong, and are we really keeping them updated?

Reply:
Dropped in cbed758

Reply:
We need them in multi-module "meta" (overview) manual pages since the parser cannot yet create that.
Reinstated with fixed URLs in 6cbeed194d924837ef9bb324871ddb3c9f2ecbcf